### DISCUSSION TITLE
2024/01/tips-to-help-yourself-stand-out-during-a-tech-job-search/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#869] [Specific Post Comments]
# 2024/01/tips-to-help-yourself-stand-out-during-a-tech-job-search/

I recently just powered through a bunch of applications for some roles I had open. As a hiring manager, I want to mention some simple things that can set you apart in your search. All of these are based on patterns I saw; way more than one candidate was doing these things.

https://seankilleen.com/2024/01/tips-to-help-yourself-stand-out-during-a-tech-job-search/

**Comment by Fnthawar** [comment]
Why not build something?

**Reply by SeanKilleen** [reply]
Great question! In my opinion: because the investment to do it well is non-trivial and the return on that investment is not guaranteed. It can end up hurting you in the process if someone doesn't think it's up to par or disagrees with your choices, or if the thing you built seems too trivial (e.g. "oh, another to-do list app").

Don't get me wrong, I love building things! And I like seeing other people build things! And I try to give lots of credit and go easy on the criticism when I see it. But, let's say I'm looking for automated testing as a day 1 skill. What if the thing they built doesn't have automated tests? Now I'm wondering if they have that skill. Or what if it's done wildly different than how I would want it to be approached? I've trained myself to try to filter out these biases, and I would turn those into open-ended interview questions & conversation. But I can imagine a lot of other organizations and hiring managers wouldn't, so it could be a double-edged sword.

This isn't to say candidates shouldn't do it or won't stand out if they do. It's just to explain why it didn't make it onto my "definitely do this" list. I was looking for practical changes people might be able to make with reasonable ROI.

**Reply by Fnthawar** [reply]
I was actually thinking about this differently.

If you build something (even if it's just a 1-2 day effort) — and especially if the target company has an API / platform you can build on top of
- you're building out your portfolio
- you're building useful skills
- if you don't get the job, you've still made forward progress on something
- you're standing out against other applicants
- you can write/tweet about your experience
- you may find a bug or be able to give feedback on the DX of the experience
- you have something to talk about in the interview
- etc

Some of the other tactics, don't have value beyond "this particular job search"

I'm trying to figure out the "how do you win, even if you lose" tactics

**Reply by SeanKilleen** [reply]
Understood. My point was that "you win even if you lose" doesn't necessarily hold true of the things you mention.

**Comment by Fnthawar** [comment]
life would be boring if people agreed on everything 🤓
